src/app/containers/AVPlayer/index.test.jsx

```diff
@@ -50,43 +50,33 @@ const AVPlayerCanonical = (
 const AVPlayerAMP = (
```

Comment:
With this const & `AVPlayerCanonical`, I'd also make it clear that the player type is different, so:

`AVPlayerCanonical` -> `AVPlayerCanonicalTV`
`AVPlayerAMP` -> `AVPlayerAMPRadio`

I'm not just suggesting this to be pernickety - I think it's important to highlight that you're changing more than just the `platform` for this fixture data.